library/time-library/src/test/java/uk/gov/gchq/gaffer/time/serialisation/RBMBackedTimestampSetSerialiserTest.java

```diff
@@ -188,8 +192,11 @@ public void shouldJSONSerialiseFloatRDM() throws IOException {
         final CustomMap deserialiseMap = JSONSerialiser.deserialise(serialise, CustomMap.class);
 
         //then
-        assertEquals(jsonMap, deserialiseMap, "The expected map from Json doesn't match");
-        assertEquals(expectedMap, deserialiseMap, "The expected map doesn't match");
+        assertThat(deserialiseMap)
+            .as("The expected map from Json doesn't match")
```

Comment:
Is was a bit odd to begin with shouldn't this really be two separate assertions as its testing if the `jsonMap` and the `desrialiseMap` are as expected e.g.

```java
assertThat(jsonMap).isEqualTo(expectedMap);
assertThat(deserialiseMap).isEqualTo(expectedMap);
```